app/views/claims/support/support_users/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for(:page_title) { @user.full_name } %>
+
+<% content_for(:before_content) do %>
+  <%= govuk_back_link href: claims_support_support_users_path %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= @user.full_name %></h1>
+
+      <%= govuk_summary_list do |summary_list| %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: User.human_attribute_name(:first_name)) %>
+          <% row.with_value(text: @user.first_name) %>
+          <% row.with_action(text: t(".change"), href: edit_claims_support_support_user_path(@user), visually_hidden_text: @user.first_name) %>
+        <% end %>
+
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: User.human_attribute_name(:last_name)) %>
+          <% row.with_value(text: @user.last_name) %>
+          <% row.with_action(text: t(".change"), href: edit_claims_support_support_user_path(@user), visually_hidden_text: @user.last_name) %>
+        <% end %>
+
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: User.human_attribute_name(:email)) %>
+          <% row.with_value(text: @user.email) %>
+          <% row.with_action(text: t(".change"), href: edit_claims_support_support_user_path(@user), visually_hidden_text: @user.email) %>
```

Comment:
When we create a user, the first and last name are used for the invite. After that, we get the information from DfE Sign in -- it's not something that can be changed. I talked to Simon about this briefly when i was in the office on Thursday. So these 'change' links should be removed.

Again, this is the 3rd time we have created this show page, so perhaps it's time to create a shared partial.

Reply:
Contextually, the partial differs from other user show pages. It only happens to show the same content in the current design.